doc/website/getting-started/overview.md

```diff
@@ -11,637 +10,290 @@ runtime and create _publishers_ and _subscribers_. The publishers send data of a
+
+### Runtime
+
+Each application that wants to use iceoryx has to instantiate its runtime, which essentially enables communication
+with RouDi.
+
+To do so, the following lines of code are required
 ```cpp
-while (keepRunning)
-{
-    while (subscriber.hasData())
-    {
-        subscriber->take()
-            .and_then([](iox::popo::Sample<const CounterTopic>& sample) {
-                CounterTopic* ptr = sample.get();
-                /* process the received data using the ptr */
-
-                /* alternatively use operator-> */
-                uint32_t counter = sample->counter;
-            })
-            .or_else([](iox::popo::ChunkReceiveResult) { /* handle the error */ });
-    }
-    // wait for new samples (either sleep or use the waitset)
-}
+iox::runtime::PoshRuntime::initRuntime("some_unique_application_name");
 ```
 
-Here we do not check whether we actually have data since we already know there is data available by calling `hasData`.
+The runtime is an object inside the user application that maps the shared memory into the user 
+application's address space. When an application terminates, the runtime cleans up all resources needed for the 
+communication with RouDi. If an application crashes, it also takes care about the clean-up. Due to our 
+[lock-free constructs](https://github.com/eclipse-iceoryx/iceoryx/tree/master/iceoryx_utils#concurrent), 
+a crash of one application does not affect the whole iceoryx system.
+
+!!! note
+    Only one runtime object per user application is allowed.
 
-### Untyped API
+### Creating service descriptions for topics
 
-The untyped API offeres similar capabilities and is hence usable in a similar way. The major difference is that
-neither publisher nor subscriber have any knowledge about the underlying type they send or receive. This means that
-the user is responsible to ensure the data is read correctly, i.e. there is no type safety guaranteed by the API
-itself.
+A ``ServiceDescription`` in iceoryx represents a topic under which publisher and subscribers can exchange data and is 
+uniquely identified by three string identifiers.
 
-#### Creating a publisher
+1. ``Group`` name
+2. ``Instance`` name
+3. ``Topic`` name
 
-When creating an untyped publisher we do not need to specify a data type as template paraemter.
+A triple consisting of such strings is called a ``ServiceDescription``. Two ``ServiceDescription``s are considered 
+matching if all these three strings are element-wise equal, i.e. group, instance and topic names are the same for both 
+of them. This means the group and instance identifier can be ignored to create different ``ServiceDescription``s. They 
+will be needed for advanced filtering functionality in the future.
 
-```cpp
-iox::popo::UntypedPublisher publisher({"Group", "Instance", "CounterTopic"});
-publisher.offer();
-```
+The service model of iceoryx is derived from AUTOSAR and is still used in the API with these names. The so called 
```

Comment:
okay, it's not gonna be changed for v1.0.0. I think it might make sense to add `... with these names(Service, Instance, Event).`, although it is in the table below.